includes/options/views/class-amp-analytics-options-serializer.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+require_once( AMP__DIR__ . '/includes/utils/class-amp-html-utils.php' );
+
+class Analytics_Options_Serializer {
```

Comment:
Let's rename this to `AMP_Options_Manager` and we can use it for other options we decide to add in the future.